samples/sampler/stories/qa/Panels.js

```diff
@@ -0,0 +1,180 @@
+				<Item onClick={goPrevious}>Previous</Item>
+				<Item onClick={goNext}>Next</Item>
+			</Panel>
+			<Panel>
+				<Header title="Third Panel" />
+				<Item onClick={goPrevious}>Previous</Item>
+			</Panel>
+		</Panels>
+	);
+};
+
+const MenuItemBase = kind({
+	name: 'MenuItem',
+
+	propTypes: {
+		css: PropTypes.object,
+		icon: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+		path: PropTypes.string
+	},
+
+	defaultProps: {
+		disabled: false
+	},
+
+	styles: {
+		className: 'menuItem'
+	},
+
+	handlers: {
+		onClick: handle(
+			forward('onClick'),
+			action('onClick')
+		)
+	},
+
+	render: ({css, ...rest}) => {
+		return (
+			<Item
+				css={css}
+				{...rest}
+				slotAfter={
+					<Icon size="small">arrowlargeright</Icon>
+				}
+			/>
+		);
+	}
+});
+
+const MenuItem = Linkable(MenuItemBase);
+
+const MainPanel = kind({
+	name: 'MainPanel',
+
+	render: (props) => (
+		<Panel spotlightId="main-panel-container" {...props}>
+			<MenuItem path="./page1">Pages</MenuItem>
+			<MenuItem path="./page2">General</MenuItem>
+		</Panel>
+	)
+});
+
+const Page1 = kind({
+	name: 'Page',
+
+	render: (props) => (
+		<Panel spotlightId="page1-container" {...props}>
+			<MenuItem path="./endPage">Page1</MenuItem>
+			<MenuItem path="./endPage">Page2</MenuItem>
+		</Panel>
+	)
+});
+
+const Page2 = kind({
+	name: 'Page',
+
+	render: (props) => (
+		<Panel spotlightId="page2-container" {...props}>
+			<MenuItem path="./endPage">General1</MenuItem>
+			<MenuItem path="./endPage">General2</MenuItem>
+		</Panel>
+	)
+});
+
+const EndPage = () => <h1>EndPage</h1>;
+
+const BreadcrumbPanelsBase  = kind({
+	name: 'Panels',
+	propTypes: {
+		onNavigate: PropTypes.func
+	},
+	render: ({onNavigate, ...rest}) => {
+		return (
+			<BreadcrumbPanels {...rest} noCloseButton onSelectBreadcrumb={onNavigate} />
+		);
+	}
+});
+
+const RoutablePanels = Routable({navigate: 'onNavigate'}, BreadcrumbPanelsBase);
+
+const RoutablePanelsApp = class extends React.Component {
+	static displayName = 'RoutablePanelsApp';
+
+	constructor (props) {
+		super(props);
+		this.state = {appPath: '/settings'};
+	}
+
+	onNavigate = ({path}) => {
+		this.setState({appPath: path});
+	};
+
+	render () {
+		return (
+			<div {...this.props}>
+				<RoutablePanels path={this.state.appPath} onNavigate={this.onNavigate} cover="partial">
+					<Route path="settings" component={MainPanel}>
+						<Route path="page1" component={Page1}>
+							<Route path="endPage" component={EndPage} />
+						</Route>
+						<Route path="page2" component={Page2}>
+							<Route path="endPage" component={EndPage} />
+						</Route>
+					</Route>
+				</RoutablePanels>
+			</div>
+		);
+	}
+};
+
+storiesOf('Panels', module)
+	.add(
+		'preserve focus',
+		() => (
+			<BasicPanels
+				noAnimation={boolean('noAnimation', Config, true)}
+				noCloseButton={boolean('noCloseButton', Config, true)}
+				orientation={select('orientation', ['horizontal', 'vertical'], Config)}
```

Comment:
If there is no specific reason for set some props, please follow the default value of Panels.
```suggestion
				noAnimation={boolean('noAnimation', Config)}
				noCloseButton={boolean('noCloseButton', Config)}
				orientation={select('orientation', ['horizontal', 'vertical'], Config)}
```